ui/src/app/pages/data/cohort-review/sidebar-content.component.tsx

```diff
@@ -294,6 +294,7 @@ export const SidebarContent = fp.flow(
       .then(({items}) => {
         this.setState({annotationDefinitions: items});
       });
+    participantStore.subscribe(participant => this.setState({participant: participant || {} as ParticipantCohortStatus}));
```

Comment:
We should make sure to unsubscribe at `componentWillUnmount` (Per @calbach's unsubscribe note)

Reply:
Addressed this on the last commit, forgot to comment.